### ISSUE TITLE
Minor problem with scrolling using the mouse wheel

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap#2271]
Using the mouse's scroll wheel I was able to create this:

![image](https://user-images.githubusercontent.com/2138696/49140135-60cbb780-f2eb-11e8-905e-8f12cdb5f834.png)

When I scroll using the scroll bar the value talk balloon goes away but not when I scroll using the mouse wheel. Shouldn't they behave the same?

**Comment by jmoenig** [comment]
Yeah, that's kinda okay. Sometimes the contents of the bubble is scrollable, such as a list watcher inside, so we want to be able to scroll that, and not make the bubble disappear.

**Comment by brianharvey** [comment]
Boy, talk about an undiscoverable feature!

**Comment by brianharvey** [comment]
No, I mean the fact that there's a difference between scroll-wheel scrolling and scrollbar scrolling.  I pretty much always use my scroll wheel; I'd never find out that the scroll bar behaves differently.
